netlify.toml

```diff
@@ -6,7 +6,7 @@ HUGO_VERSION = "0.53"
 
 [build]
 publish = "public"
-command = "gulp sass && hugo"
+command = "gulp sass && npm run data-medium && hugo"
```

Comment:
Netlify will download the feed. You can do it manually with `npm run data-medium`. This should probably go into the readme or combined into package.json scripts.